docker-compose.ci.yml

```diff
@@ -15,6 +15,10 @@ services:
      - LUMEN_ENCRYPTION_KEY=secret
      - LUMEN_SENTRY_BACKEND_DSN=https://xxxx@sentry.io/111111
      - LUMEN_SENTRY_CLIENT_DSN=https://xxxx@sentry.io/111111
+     - LUMEN_SENTRY_BACKEND_DSN=${LUMEN_SENTRY_BACKEND_DSN}
+     - LUMEN_SENTRY_SERVER_NAME="green"
+     - LUMEN_SENTRY_RELEASE=${TRAVIS_COMMIT}
```

Comment:
Not sure if these values are needed or useful 🤔

Reply:
I mean : LUMEN_SENTRY_SERVER_NAME, LUMEN_SENTRY_RELEASE, LUMEN_SENTRY_ENVIRONMENT